doc/dev/workflow.md

```diff
@@ -0,0 +1,63 @@
+# Overview
+
+This document describes the general developement workflow for
+developing and working with Mixer in the broader context of Istio.
+This document includes information on building Mixer properly and
+then building the associated containers in a local environment
+prior to prow updating the istio/istio dependency repositories.
+
+## Prepare tree
+
+Prepare the Mixer repository and Go environment variables as
+specified in the 
+[contribution guidelines](https://github.com/istio/istio/blob/master/CONTRIBUTING.md).
+
+## Prepare a registry account on gcr.io
```

Comment:
IMO, this should be "Prepare a container registry" as other services like docker.io could be used as well with the --hub param in mixer's bin/publish-docker-images.sh and pilot's bin/push-docker scripts.

Reply:
fixed.

Reply:
fixed.